.github/workflows/test-environment.yml

```diff
@@ -10,6 +10,11 @@ on:
         description: |
           Name with letters, numbers, hyphens; start with a letter. Max 20 chars. e.g., 'my-env-123'
         required: true
+      serverless_mode:
+        description: "Deploy a serverless project instead of an ESS deployment"
+        type: boolean
+        required: true
+        default: false
```

Comment:
we can use [choice](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/) as an input type for the deployment type